libraries/AP_ICEngine/AP_ICEngine.cpp

```diff
@@ -255,6 +255,11 @@ void AP_ICEngine::update(void)
     }
 #endif
 
+    if (option_set(Options::NO_RUNNING_WHILE_DISARMED) && !hal.util->get_soft_armed()) {
```

Comment:
This is fine, but why didn't you put it up next to the other option_set() check?

Reply:
Totally could have, I just tacked it on the end, because that's where I'd had it in a previous branch. Does it matter?

Reply:
would be nice to group them together. Pretty please?